pynb.py

```diff
@@ -17,6 +18,10 @@
 import paramiko
 SSH_PORT = 22
 DEFAULT_PORT = 8888
+DEFAULT_CPU = 8
```

Comment:
8 cpus is probably overkill, 1 cpu for default and 4G mem should be all that's needed if the user needs more they should 1 realise this and 2 be able to request it.